src/build/disagg/disagg_region.jl

```diff
@@ -193,10 +193,13 @@ This function aggregates final demand categories into national consumption (`C`)
 (`G`), and investment (`I`) demand.
 """
 function _disagg_fdcat!(d::Dict)
-    if !(:fdcat in propertynames(d[:fd0]))
-        x = Map(joinpath("crosswalk","fd.csv"), [:fd], [:fdcat], [:fd], [:fdcat], :inner)
+    if "pce" in d[:fd0][:,:fd]
+        x = [
+            Map(joinpath("crosswalk","fd.csv"), [:fd], [:fdcat], [:fd], [:fd], :inner),
+            Combine("sum",propertynames(d[:fd0])),
+        ]
         d[:fd0] = edit_with(d[:fd0], x)
-        d[:fd0] = combine_over(d[:fd0], :fd)
+        # d[:fd0] = combine_over(d[:fd0], :fd)
     end
```

Comment:
**For future** If we map `fd -> fdcat` upfront (during data stream), we won't need to here.